test/metabase/api/search_test.clj

```diff
@@ -81,10 +80,17 @@
    (t2/select-one [:model/Table [:name :table_name] [:schema :table_schema] [:description :table_description]]
                   :id (mt/id :checkins))))
 
-(defn- sorted-results [results]
-  (->> results
-       (sort-by (juxt (comp (var-get #'scoring/model->sort-position) :model)))
-       reverse))
+(defn- clean-result [result]
+  (dissoc (u/remove-nils result) :bookmark :database_id :model_id :model_index_id :table_id :dataset_query :last_editor_id :last_edited_at
```

Comment:
Forgot about skipping these fields. Laziness when first trying to get the tests passing as a basic level.

We should look into if any of these need to be preserved.